### ISSUE TITLE
Module ft_mha not found

### ISSUE THREAD

**Issue by Manuel030** [alpa-projects/alpa#771]
**Please describe the bug**
When trying to load opt llm serving as in the examples, I get the following error.

```
python3 textgen.py --model alpa/opt-2.7b

Traceback (most recent call last):
  File "/home/ubuntu/Projects/opt-alpa/alpa/examples/llm_serving/model/opt_model_1d.py", line 30, in <module>
    from ft_mha import fused_mmha
ModuleNotFoundError: No module named 'ft_mha'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "textgen.py", line 7, in <module>
    from llm_serving.model.wrapper import get_model
  File "/home/ubuntu/Projects/opt-alpa/alpa/examples/llm_serving/model/wrapper.py", line 20, in <module>
    from llm_serving.model import opt_model, bloom_model, opt_model_1d
  File "/home/ubuntu/Projects/opt-alpa/alpa/examples/llm_serving/model/opt_model_1d.py", line 32, in <module>
    raise RuntimeError("Please install ft_mha to use 1D OPT model.")
RuntimeError: Please install ft_mha to use 1D OPT model.

```
**Please describe the expected behavior**
I should be able to load the model, or the module `ft_mha` should be made available.

**System information and environment**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04, docker): Ubuntu 20.04
- Python version: 3.8
- CUDA version: 11.3
- Alpa version: 0.2.1

Thank you for the great library!

**Comment by merrymercy** [comment]
Hi, you can simply comment out this line

@zhisbug Please fix this bug and do not import `opt_model_1d.py` when using `opt_model.py` only

**Comment by zhisbug** [comment]
@Manuel030 fixed by #768 . Let us know if there is still a problem.
